BPOR/BPOR.Rms/Views/Researcher/SubmitStudy.cshtml

```diff
@@ -10,15 +10,7 @@
     <div class="govuk-grid-column-two-thirds govuk-grid-column-one-half-from-desktop">
 
         <form asp-action="SubmitStudy" back-link-for="Step">
-            @if (ViewData["Step"] is not null && ViewData["TotalSteps"] is not null)
-            {
-                var step = ViewData["Step"] as int?;
```

Comment:
Check where else we are using `ViewData["Step"]` and `ViewData["TotalSteps"]` if it is not read _anywhere_ anymore, remove all the locations that are setting this these values.

e.g.
https://github.com/PA-NIHR-CRN/dte-study-api/pull/115/files#diff-a27974a40fdd743b11a3a7b0675ac1d489fdd14c19f875f6c8487e7c0362c0c6R174